INSTALL.md

```diff
@@ -33,10 +33,10 @@ _NOTE:_ On CentOS/RHEL 6 or less, you may skip `libseccomp-devel`.
 
 This is one of several ways to [install and configure golang](https://golang.org/doc/install).  The CentOS/RHEL instructions above already installed it so this method is not needed there.
 
-First, download the Golang archive to `/tmp/`, then extract the archive to `/usr/local`.
+First, download the Golang archive to `/tmp`, then extract the archive to `/usr/local`
 
 ```
-$ export VERSION=1.12.6 OS=linux ARCH=amd64  # change this as you need
+$ export VERSION=1.13 OS=linux ARCH=amd64  # change this as you need
```

Comment:
this one is interesting: we are not really testing with 1.13 yet.

There's an issue with modules and 1.13, namely that `go mod vendor` will remove certain unnecessary files (which is good) in a way that is different to what 1.12 did (which is bad). This interacts badly with checks in our CI pipeline.

For starters, we would need to ask developers to use 1.13, otherwise if they add or update a module they will end up (maybe) missing files wrt to 1.12 when they run `go mod vendor`. This means CI will fail.

If we ask developers to use 1.13, then there's the question of what do we do about rpms, as they seem to be building using the version provided by the distro. I think that's partly the reason why we run CI with 1.11 and 1.12.

We could do the vendor checks with 1.13 (and ask developers to use 1.13, too) and keep 1.11 and 1.12 in the CI pipeline, without checking the vendored code there. I'm ok with that. But we need to bring 1.13 into CI first.

Reply:
Agree, is not a priority to merge. just wanted to point out the outdated INSTALL docs. let's mark this PR as blocked by the go 1.13 issue

Reply:
@mem I'd been playing and testing 3.4 with Go 1.13, and ... it works....as the long known story...works on my machine....

Reply:
golang-1.13 is now the default in EPEL.  It works for me.

Reply:
`1.13` is good for building - but if someone does dev from and does a `go mod vendor` I believe we still have the CI issues @mem is mentioning. If this is the case, can you set it back to `1.12.10` @ArangoGutierrez to avoid confusion there?